### PR TITLE
Jagruti

### PR DESCRIPTION
**Pull request by jagrutipatel199320**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# AccessLocator
+# AccessLocator test git
```
Comment (joshi1983) [inline README.md]: Completely remove these commits as they are not related to your issue:
image

I would do this with the following command sequence:
1. ```git rebase -i master```
1. Remove the lines that look like "pick 123some hexidecimal6789 readme file change
1. Save and exit.
1. ```git push origin jagruti -f```
```

File: app/app/Http/Controllers/LocationManagementController.php [hunk 2]
```diff
@@ -118,7 +123,16 @@ public function addNewLocation()
 		$view_data['locations'] = json_encode($this->getLocationsNear($view_data['location']->longitude, $view_data['location']->latitude));
 		return view('pages.location_management.add_new_location', $view_data);
 	}
+	public function editLocation($location_id)
+	{
+		$view_data = $this->getLocationAddViewData();
+		$view_data['location'] = DB::table('location')->where("id", '=', trim($location_id))->first();
+		//print_r($test->name);exit;
```
Comment (joshi1983) [inline app/app/Http/Controllers/LocationManagementController.php]: Remove this commented line of code.

File: app/app/Http/Controllers/LocationManagementController.php [hunk 3]
```diff
@@ -211,11 +226,103 @@ public function addNewLocationSave(Request $request)
...
+		$location_group_id = Input::get('location_group_id');
+		if ( !is_numeric($location_group_id) ) {
+			$location_group_id = null; // convert things like '-' to null.
+		}
+		else {
+			$location_group_id = intval($location_group_id);
+		}
+		$location = $view_data['location'];
+		$location->location_group_id = $location_group_id;
+		//$view_data['locations'] = json_encode($this->getLocationsNear($location->longitude, $location->latitude));
```
Comment (joshi1983) [inline app/app/Http/Controllers/LocationManagementController.php]: Remove this commented code.  There's no need to keep it around when it doesn't do anything useful.

File: app/resources/views/pages/location_management/edit_location.blade.php [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+					<select id="location_group_id" name="location_group_id">
+						<option value="none">None</option>
+					@foreach ($location_groups as $location_group) 
+						<option value="{{ $location_group->id }}" @if ( $location->location_group_id === $location_group->id )
+							selected
+						@endif>{{ $location_group->name }}</option>
+					@endforeach
+						<option value="-">Other</option>
+					</select>
+					<button class="btn btn-primary" type="submit">Add</button>
```
Comment (joshi1983) [inline app/resources/views/pages/location_management/edit_location.blade.php]: Reword "Add" to "Save".  "Add" is misleading since we're only updating an existing location.

File: app/routes/web.php [hunk 5]
```diff
@@ -5,5 +5,42 @@
 Route::get('terms-of-use', function() { return View::make('pages.terms_of_use'); });
 Route::get('privacy-policy', function() { return View::make('pages.privacy_policy'); });
 Route::get('faq', 'FaqController@index');
```
Comment (joshi1983) [inline app/routes/web.php]: We can tackle this problem with routes/web.php after you get those "readme file change" commits removed.  I'll give tips regarding this after you do that.

You just added the following 2 lines to web.php, right?
```
Route::get('edit-my-location/{location_id}', 'LocationManagementController@editLocation');
Route::post('edit-location', 'LocationManagementController@editLocationSave');
```

Comment (jagrutipatel199320) [reply app/routes/web.php]: yes

File: app/app/Http/Controllers/LocationManagementController.php (hunk 2 above)
Comment (jagrutipatel199320) [reply app/app/Http/Controllers/LocationManagementController.php]: done

File: app/app/Http/Controllers/LocationManagementController.php (hunk 3 above)
Comment (jagrutipatel199320) [reply app/app/Http/Controllers/LocationManagementController.php]: done

File: app/resources/views/pages/location_management/edit_location.blade.php (hunk 4 above)
Comment (jagrutipatel199320) [reply app/resources/views/pages/location_management/edit_location.blade.php]: done

Comment (joshi1983) [reply app/resources/views/pages/location_management/edit_location.blade.php]: No.  It looks like you reworded "Add" to "Save" in the wrong file.

You set "Save" in add_new_location.blade.php when it should have been "edit_location.blade.php".

File: app/resources/views/pages/location_management/location_created.blade.php [hunk 6]
```diff
@@ -1,9 +1,14 @@
 @extends('layouts.default')
 @section('content')
 <div class="location-created">
+	
+	@if(!empty($action))
+	<h1 class="text-center">Location Modified</h1>
+	<p class="text-center">The location has been Modified.<p>
```
Comment (joshi1983) [inline app/resources/views/pages/location_management/location_created.blade.php]: Lower case "m" for "The location has been modified." would look a little better.

File: app/app/Http/Controllers/LocationManagementController.php [hunk 7]
```diff
@@ -211,11 +225,102 @@ public function addNewLocationSave(Request $request)
...
+		}
+		else {
+			$custom_validation_failed = true;
+		}
+		if ($custom_validation_failed) {
+			$view_data['errors'] = $validator->errors();
+			return view('pages.location_management.add_new_location', $view_data);
+		}
+
+		//$location->creator_user_id = $user->id;
```
Comment (joshi1983) [inline app/app/Http/Controllers/LocationManagementController.php]: These commented lines of code can be removed.

File: app/resources/views/pages/location_management/edit_location.blade.php (hunk 4 above)
Comment (jagrutipatel199320) [reply app/resources/views/pages/location_management/edit_location.blade.php]: i dont need edit_location.blade.php file. its extra file i am removing that. as i have written code in add_location.blade.php for edit, i have just set flag and different functionality for both action add/ edit

File: app/resources/views/pages/location_management/location_created.blade.php (hunk 6 above)
Comment (jagrutipatel199320) [reply app/resources/views/pages/location_management/location_created.blade.php]: done

File: app/app/Http/Controllers/LocationManagementController.php (hunk 7 above)
Comment (jagrutipatel199320) [reply app/app/Http/Controllers/LocationManagementController.php]: done

**Comment by joshi1983** [comment]
It looks like some .DS_Store files creeped in again.  Can you remove them?

image

**Comment by joshi1983** [comment]
I'm rebasing on my end to get these changes in with a pull request likely to happen tonight.
